src/services/chat/prepopulated.js

```diff
@@ -39,28 +38,22 @@ module.exports = async (id, users) => {
 
         const textTargetUser = `${ownUser.username} started following you. Send them a message now`;
         const textOwnUser = `You started following ${user.username}. Send them a message now.`;
-        await chat.addMembers([id], {
-          text: textOwnUser,
-          user_id: id,
-          only_to_user_show: id,
-          disable_to_user: false,
-          channel_role: 'channel_moderator',
-          is_add: true,
-          system_user: id,
-          is_from_prepopulated: true,
-          other_text: textTargetUser
-        });
-        await chat.addMembers([user.user_id], {
-          text: textTargetUser,
-          user_id: user.user_id,
-          only_to_user_show: false,
-          disable_to_user: id,
-          channel_role: 'channel_moderator',
-          is_add: false,
-          system_user: id,
-          is_from_prepopulated: true,
-          other_text: textOwnUser
-        });
+        await chat.sendMessage(
+          {
+            user_id: id,
+            text: textOwnUser,
+            only_to_user_show: id,
+            disable_to_user: false,
+            is_from_prepopulated: true,
+            other_text: textTargetUser,
+            own_text: textOwnUser,
+            system_user: id,
+            is_system_message: true
+          },
+          {
+            skip_push: true
+          }
+        );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In this hunk, the `chat.addMembers` method calls have been replaced with a single `chat.sendMessage` call. This seems to simplify the process of sending system messages to users. However, it's important to ensure that this change doesn't affect the functionality of the application. 

If the `addMembers` method was doing something crucial (like setting the `channel_role` or handling different texts for different users), then this change might introduce bugs. Without knowing more about the `StreamChat` library and how it's used in your application, I can't say for sure whether this is an issue. You should verify that `sendMessage` is an appropriate replacement for `addMembers` in this context.


<!-- This is an auto-generated comment by OSS CodeRabbit -->